### PR TITLE
Add | Code Example in the Readme for usage with Functional Components …

### PR DESCRIPTION
**Pull request by ZeroPie**
Add | Code Example in the Readme for usage with Functional Components and Hooks
Add | Section Title for Usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ transpiled es6 classes rather than `createClass` as of v6.
 ## Sections covered in this README
 
 * [Installation](#installation)
+* [Usage](#usage)
```
Comment (Pomax) [inline README.md]: we might as well also get those nested entries in, as it'll let people laser-focus on what they're looking for. Probably indented?

File: README.md [hunk 2]
```diff
@@ -79,6 +108,8 @@ var MyComponent = onClickOutside(
 );
 ```
 
+---
```
Comment (Pomax) [inline README.md]: horizontal rules usually mean there's a missing heading, let's replace that `---` with a heading `### ensuring there's a click handler"?

File: README.md (hunk 1 above)
Comment (ZeroPie) [reply README.md]: Good idea, usage headings are now indented under Usage)

**Comment by Pomax** [comment]
sweet, thanks for the help!

**Comment by Pomax** [comment]
how about I also merge it. yes, great idea.

**Pull request merged**